Framework/include/QualityControl/PostProcessingConfig.h

```diff
@@ -42,8 +42,6 @@ struct PostProcessingConfig : public o2::quality_control::core::UserCodeConfig {
   std::string kafkaTopicAliECSRun;
   core::Activity activity;
   bool matchAnyRunNumber = false;
-  bool critical;
-  bool validityFromLastTriggerOnly = false;
```

Comment:
bad merge? i wouldn't expect the latter to disappear

Reply:
yes, bad merge